src/cljs/main/broadfcui/components/script_loader.cljs

```diff
@@ -14,10 +14,11 @@
   :on-error
   :on-load
   :path
-  :attributes (optional) - additional html attributes to be placed on the script tag"
+  :attributes (optional) - additional html attributes to be placed on the script tag
+  :allow-cache? (optional) - pass true to avoid reloading on every new build"
```

Comment:
Noob question ... where would this be set? Is this a config or a jenkins job kind of option?

Reply:
This would happen whenever you invoke the script loader - in its props, you can pass `:allow-cache? true`